provider.go

```diff
@@ -182,6 +189,19 @@ func (wp *WasmcloudProvider) listenForActor(actorID string) {
 	wp.natsSubscriptions[actorID] = actorsub
 }
 
+func (wp *WasmcloudProvider) validateProviderInvocation(invocation core.Invocation) error {
+	// todo validate claims issuer is included in cluster issuers
```

Comment:
I'd recommend filing an issue so we don't lose track of this TODO

Reply:
Added issue https://github.com/wasmCloud/provider-sdk-go/issues/4